projects/sunbird-video-player/src/lib/components/video-player/video-player.component.scss

```diff
@@ -196,4 +196,8 @@
 
 div[data-marker-key] {
   margin-left: 0.7% !important;
+}
+
+.vjs-texttrack-settings {
+  display: none;
```

Comment:
Add the comments why are you disabled. Add TODO comment

Reply:
Done